### PR TITLE
resource_control: add more info about sql ru consumption

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
This PR depends on #15935

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):https://github.com/pingcap/docs-cn/pull/15935

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-resource-control.md [hunk 1]
```diff
@@ -438,6 +434,85 @@ Runaway Query 是指执行时间或消耗资源超出预期的查询（仅指 `S
...
+你也可以通过 [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-消耗) 语句获取到 SQL 执行时所消耗的 RU。注意 RU 的大小会受缓存的影响（比如[下推计算结果缓存](/coprocessor-cache.md)），多次执行同一条 SQL 所消耗的 RU 可能会有不同。因此这个 RU 值并不代表每次执行的精确值，但可以作为估算的参考。
+
+#### 慢查询及对应的系统表
+
+在开启资源管控时，TiDB 的[慢查询日志](/identify-slow-queries.md)以及对应系统表 [`INFORMATION_SCHEMA.SLOW_QUERY`](/information-schema/information-schema-slow-query.md) 中均包含对应 SQL 对应的资源组以及 RU 消耗等相关信息。
+
+#### `statements_summary`
+
+TiDB 的系统表 [`INFORMATION_SCHEMA.statements_summary`](/statement-summary-tables.md#statements_summary) 中保存了 SQL 语句归一化聚合后的各种统计信息，可以用于查看分析各个 SQL 语句的执行性能。其中也包含资源管控相关的统计信息，包括资源组名、RU 消耗、等待可用 RU 的耗时等信息。具体请参考[`statements_summary` 字段介绍](/statement-summary-tables.md#statements_summary-字段介绍)。
+
```
Comment (hfxsd) [inline tidb-resource-control.md:488]: 相关内容在这个 PR 里 https://github.com/pingcap/docs-cn/pull/15935/files#diff-ef4f47e6759b3396f549d89875cdeb21a4d3c7205aa21658cc90e384cf17d7e4

File: tidb-resource-control.md [hunk 2]
```diff
@@ -438,6 +434,85 @@ Runaway Query 是指执行时间或消耗资源超出预期的查询（仅指 `S
...
+    返回结果中的 `ru_consumption` 即为执行此 SQL 语句消耗的 RU。
+
+#### `EXPLAIN ANALYZE`
+
+你也可以通过 [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-消耗) 语句获取到 SQL 执行时所消耗的 RU。注意 RU 的大小会受缓存的影响（比如[下推计算结果缓存](/coprocessor-cache.md)），多次执行同一条 SQL 所消耗的 RU 可能会有不同。因此这个 RU 值并不代表每次执行的精确值，但可以作为估算的参考。
+
+#### 慢查询及对应的系统表
+
+在开启资源管控时，TiDB 的[慢查询日志](/identify-slow-queries.md)以及对应系统表 [`INFORMATION_SCHEMA.SLOW_QUERY`](/information-schema/information-schema-slow-query.md) 中均包含对应 SQL 所属的资源组以及 RU 消耗等相关信息。
```
Comment (songrijie) [inline tidb-resource-control.md:483]:
```suggestion
在开启资源管控时，TiDB 的[慢查询日志](/identify-slow-queries.md)以及对应系统表 [`INFORMATION_SCHEMA.SLOW_QUERY`](/information-schema/information-schema-slow-query.md) 中均包含对应 SQL 所属的资源组，等待可用 RU 的耗时，以及真实 RU 消耗等相关信息。
```

**Review by songrijie — APPROVED** [review]
rest LGTM

File: tidb-resource-control.md [hunk 3]
```diff
@@ -438,6 +434,85 @@ Runaway Query 是指执行时间或消耗资源超出预期的查询（仅指 `S
...
+
+### 查看 SQL 消耗的 RU
+
+#### 查询系统变量 `tidb_last_query_info`
+
+TiDB 提供系统变量 [`tidb_last_query_info`](/system-variables.md#tidb_last_query_info-从-v4014-版本开始引入)，记录上一条 DML 语句执行的信息，其中包含 SQL 执行消耗的 RU。
+
+使用示例：
+
+1. 首先执行 `UPDATE`` 语句：
```
Comment (hfxsd) [inline tidb-resource-control.md:449]:
```suggestion
1. 首先执行 `UPDATE` 语句：
```

File: tidb-resource-control.md [hunk 4]
```diff
@@ -438,6 +434,92 @@ Runaway Query 是指执行时间或消耗资源超出预期的查询（仅指 `S
 
 3. 将 TiFlash 参数 [`enable_resource_control`](/tiflash/tiflash-configuration.md#配置文件-tiflashtoml) 设为 `false`，关闭 TiFlash 资源管控。
 
+## RU 消耗的可观测性
```
Comment (ran-huang) [inline tidb-resource-control.md:437]:
```suggestion
## 查看 RU 消耗
```

File: tidb-resource-control.md [hunk 5]
```diff
@@ -438,6 +434,92 @@ Runaway Query 是指执行时间或消耗资源超出预期的查询（仅指 `S
...
+- 慢查询及对应的系统表
+- `statements_summary`
+
+#### 使用系统变量 `tidb_last_query_info` 查询执行上一条 SQL 语句的 RU 消耗
+
+TiDB 提供系统变量 [`tidb_last_query_info`](/system-variables.md#tidb_last_query_info-从-v4014-版本开始引入)，记录上一条 DML 语句执行的信息，其中包含 SQL 执行消耗的 RU。
+
+使用示例：
+
+1. 首先执行 `UPDATE` 语句：
```
Comment (ran-huang) [inline tidb-resource-control.md:456]:
```suggestion
1. 执行 `UPDATE` 语句：
```

File: tidb-resource-control.md [hunk 6]
```diff
@@ -438,6 +434,92 @@ Runaway Query 是指执行时间或消耗资源超出预期的查询（仅指 `S
...
+    ```sql
+    UPDATE sbtest.sbtest1 SET k = k + 1 WHERE id = 1;
+    ```
+
+    ```
+    Query OK, 1 row affected (0.01 sec)
+    Rows matched: 1  Changed: 1  Warnings: 0
+    ```
+
+2. 然后通过查询系统变量 `tidb_last_query_info`，查看上条执行的语句的相关信息：
```
Comment (ran-huang) [inline tidb-resource-control.md:467]:
```suggestion
2. 通过查询系统变量 `tidb_last_query_info`，查看上条执行的语句的相关信息：
```

**Review by ran-huang — APPROVED** [review]

**Review by nolouch — APPROVED** [review]
lgtm.

**Review by hfxsd — APPROVED** [review]

**Pull request merged**